generator/sbpg/targets/resources/sbp_messages_template.rs

```diff
@@ -92,6 +92,11 @@ impl super::SBPMessage for (((m.identifier|camel_case))) {
     fn set_sender_id(&mut self, new_id: u16) {
         self.sender_id = Some(new_id);
     }
+
+    fn to_frame(&self) -> std::result::Result<Vec<u8>, crate::framer::FramerError> {
+        let trait_object: &dyn super::SBPMessage = self as &dyn super::SBPMessage;
```

Comment:
I think this is a bit cleaner?
```suggestion
        let trait_object = self as &dyn super::SBPMessage;
```